### DISCUSSION TITLE
Decoder Statistics

### DISCUSSION THREAD

**Discussion by TheSashmo** [CESNET/UltraGrid#161] [Q&A]
Video dec stats (cumulative): 3377399 total / 3377344 disp / 0 drop / 1513 corr / 55 missing.

What is the corr mean?  Corrected, right?  What is it correcting if I have NO FEC, or NO other ways to recover packets?

Additionally, is there a way to get the decoder stats through the control API?

**Comment by TheSashmo** [comment]
But the SSRC shows:
SSRC 0b4d78ee: 1792/1792 packets received (100%), 0 lost, max loss 0.
SSRC 1d4af6b3: 2048/2048 packets received (100%), 0 lost, max loss 0.

So what am I doing wrong here?

**Reply by MartinPulec** [reply]
It is corrupted, sorry for the abbreviations, it is to reduce the line size if possible. The frame values are cumulative for the UltraGrid lifetime, the further is statistics updated every 5 statistics.

**Reply by TheSashmo** [reply]
Well that makes A LOT OF SENSE.... Sorry for the obvious question.  My fault.
